docs/conf.py

```diff
@@ -71,7 +71,7 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', 'architecture']
```

Comment:
directories to ignore from build, so stuff we don't want rendered in the docs yet, isn't